src/shapes/Object/InteractiveObject.ts

```diff
@@ -149,11 +149,15 @@ export class InteractiveFabricObject<
   }
 
   /**
-   * Determines which corner has been clicked
+   * Determines which corner is under the mouse cursor, represented by `pointer`.
+   * This function is return a corner only if the object is the active one.
+   * This is done to avoid selecting corner of non active object and activating transformations
+   * rather than drag action. The default behaviour of fabricJS is that if you want to trasform
+   * an object, first you select it to show the control set
    * @private
    * @param {Object} pointer The pointer indicating the mouse position
    * @param {boolean} forTouch indicates if we are looking for interaction area with a touch action
-   * @return {String|Boolean} corner code (tl, tr, bl, br, etc.), or false if nothing is found
+   * @return {String|Boolean} corner code (tl, tr, bl, br, etc.), or 0 if nothing is found.
```

Comment:
imho this should be always a string, it makes much more sense. empty string if nothing is found.
Not touching it now

Reply:
I touched it and reverted to keep scope. And there is that interaction PR that changes this.